sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -105,6 +105,7 @@ statement
         ADD COLUMNS '(' columns=colTypeList ')'                        #addTableColumns
     | ALTER (TABLE | VIEW) from=tableIdentifier
         RENAME TO to=tableIdentifier                                   #renameTable
+    | RENAME (TABLE | VIEW) from=tableIdentifier TO to=tableIdentifier #renameTable
```

Comment:
Thanks for your contribution! I know, MySQL has both this syntax support and ALTER TABLE ... RENAME... 

In the current stage, we are not randomly adding more syntax to Spark SQL for avoiding extra complexity. In the future, we might support different dialects.

Reply:
Thanks a lot!